tests/common/features/value_type.rs

```diff
@@ -0,0 +1,26 @@
+use sea_orm::entity::prelude::*;
+use sea_orm_macros::DeriveValueType;
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel)]
+#[sea_orm(table_name = "value_type")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
+    pub number: Integer,
+}
+
+#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
+pub enum Relation {}
+
+impl ActiveModelBehavior for ActiveModel {}
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveValueType)]
+pub struct Integer(pub i32);
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveValueType)]
+#[sea_orm(column_type = "String(Some(1))", array_type = "String")]
+pub struct StringVec(pub Vec<String>);
```

Comment:
Great work. I think it looks really good. One question, is `array_type` required here? What if we omit it?

Can we actually change the test case in `tests/common/features/json_vec.rs` to use this `DeriveValueType` instead of the manual implementation?

Just want to make sure we have had end to end tests.